accounts/models.py

```diff
@@ -260,3 +260,12 @@ def create_user_profile(sender, instance, created, **kwargs):
         profile.save()
 
 post_save.connect(create_user_profile, sender=User)
+
+
+class Donation(models.Model):
+    email = models.CharField(max_length=255)
```

Comment:
Could we not link directly to the profile that is doing the donation? I.e. pass the Freesound user ID to Paypal when the donation is being done... We could still ask the user if she prefers to be anonymous or not.